src/util/const.ts

```diff
@@ -1,6 +1,6 @@
-import { Network as NetworkFromEthers } from '@ethersproject/networks';
+import {Network as NetworkFromEthers} from '@ethersproject/networks';
```

Comment:
Did you run `yarn format` before submitting the PR?

Reply:
nope good call, I pushed up the formatting changes just now